src/integration/pages/storyPage/ampTests.js

```diff
@@ -0,0 +1,22 @@
+import { runCoreAmpTests, runAmpAnalyticsTests } from '../../common';
+
+export default () => {
+  runCoreAmpTests();
+  runAmpAnalyticsTests();
+
+  describe('Social Embeds', () => {
+    const hasRichInstagramEmbed = !!document.querySelector(
+      'iframe.instagram-media',
+    );
+
+    if (hasRichInstagramEmbed) {
+      it("I can see the AMP component's JavaScript", () => {
```

Comment:
Include Insta in the test description?

Reply:
Please see https://github.com/bbc/simorgh/pull/6302#discussion_r419551117.